### PR TITLE
Including an option to choose whether use SSL or not

### PR DESCRIPTION
**Pull request by hugomn**
Users cans choose in config.yml wheter bundle will use PostMark https url or not.

### CODE REVIEW

File: Postmark/Message.php [hunk 1]
```diff
@@ -110,10 +117,11 @@ class Message
      * @param string     $from_email
      * @param string     $from_name
      */
-    public function __construct(HTTPClient $client, $from_email, $from_name = null)
+    public function __construct(HTTPClient $client, $from_email, $from_name = null, $use_ssl)
```
Comment (miguel250) [inline Postmark/Message.php]: Can you default $use_ssl to true?

Comment (hugomn) [inline Postmark/Message.php]: Done! ;)

**Comment by miguel250** [comment]
Thx for the pull request

**Comment by hugomn** [comment]
You're welcome. I'll keep evoluating PostMark integration. ;)

**Pull request merged**